geolocator_windows/pubspec.yaml

```
@@ -0,0 +1,28 @@
+name: geolocator_windows
+description: Geolocation Windows plugin for Flutter. This plugin provides the Windows implementation for the geolocator.
+version: 1.0.0
```

Please update this version to `0.1.0` which will indicate this is a pre-release version of the plugin which is still under development:
```suggestion
version: 0.1.0
```